packages/framework/client-logger/app-insights-logger/src/fluidAppInsightsLogger.ts

```diff
@@ -104,9 +137,52 @@ export class FluidAppInsightsLogger implements ITelemetryBaseLogger {
 		return shouldSendEvent;
 	}
 
+	/**
+	 * Checks if a given telemetry event conforms to any of the provided {@link TelemetryFilter} rules.
+	 *
+	 * - If a {@link TelemetryFilter} specifies both a `category` and a `namespace`, the event must match both.
```

Comment:
Note: this won't appear in generated comments the way you expect. It's an unfortunate symptom of how TSDoc handles hard line breaks as soft breaks. In order for these to appear as a list in hover-over / generated API docs, there needs to be a blank line between each list item.

https://github.com/microsoft/FluidFramework/wiki/TSDoc-Guidelines#line-breaks

Reply:
Updated.